ibis/expr/types.py

```diff
@@ -233,6 +233,9 @@ def equals(self, other, cache=None):
     def _root_tables(self):
         return self.op().root_tables()
 
+    def get_key(self):
```

Comment:
I would rather you just use `_key` above; this is essentially global api you are adding here)

Reply:
Okay I can definitely do that instead